### PR TITLE
More linting

### PR DESCRIPTION
**Pull request by elibarzilay**
With some more TS stuff.

### CODE REVIEW

**Comment by elibarzilay** [comment]
@jablko, does this make sense as a continuation of what you started?

(I got back the `{}`s I disliked, because the rule sounds to useful to give up completely, and adding a one-line exception is too much noise to be worth it...)

**Review by jablko — APPROVED** [review]
Yes indeed!

I opened a pull request to pull my changes into yours, but I'm happy with this PR either way :+1:

File: list-project-columns.js [hunk 1]
```diff
@@ -11,6 +11,7 @@ const headers = {
     "accept": "application/vnd.github.inertia-preview+json"
 };
 
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const projects = r('https://api.github.com/repos/DefinitelyTyped/DefinitelyTyped/projects', { headers }, (err, data) => {
```
[the remark this answers is not in the document]
Comment (elibarzilay) [reply list-project-columns.js:15]: I don't really know what the intention here was, but since it's mostly unused, I'll just do this instead...

**Comment by elibarzilay** [comment]
@jablko, After reading through some of your comments, I saw that your PR-for-this-PR addresses them.  So I'll assume that I can ignore all of your comment here and instead comment on your PR -- is that correct?

**Comment by jablko** [comment]
@elibarzilay Yup the GitHub review just wasn't adequate for some of the suggestions so the comments are here and the changes are there. Maybe I should've started reviewing your commit from that PR? I'm still learning!

**Comment by elibarzilay** [comment]
Nah, a PR for a PR makes sense to me, -- but TBH, I haven't seen many people who are comfortable with that... 😃

**Comment by jablko** [comment]
Thanks! I've restored the `_unused` arguments.

**Pull request merged**